editor/src/components/canvas/canvas-strategies/strategies/convert-to-absolute-and-move-strategy.spec.browser2.tsx

```diff
@@ -708,86 +735,224 @@ describe('Convert to absolute/escape hatch', () => {
     const currentStrategy = renderResult.getEditorState().strategyState.currentStrategy
     expect(currentStrategy).toEqual(ConvertToAbsoluteAndMoveStrategyID)
   })
+  ;(['flex', 'flow'] as const).forEach((parentLayoutSystem) =>
```

Comment:
I recommend reading the tests zero-based, ignore the diff and just look at the tests because I significantly rewrote them.

Reply:
Nah